src/AppBundle/Repository/RegisteredUserRepository.php

```diff
@@ -5,7 +5,138 @@
 
 use Doctrine\ORM\EntityRepository;
 
+use AppBundle\Entity\RegisteredUser;
+
 class RegisteredUserRepository extends EntityRepository
 {
-
-}
\ No newline at end of file
+    // /**
+    //  * @var array
```

Comment:
@ramonchristophermorales can you please make sure to remove redundant codes? (next time don't worry about it this time all good)